{
  "comments": [
    {
      "unresolved": false,
      "key": {
        "uuid": "37c46a17_05cbf468",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 1
      },
      "lineNbr": 0,
      "author": {
        "id": 1290487
      },
      "writtenOn": "2020-11-03T20:19:13Z",
      "side": 1,
      "message": "Thanks for finding the issue Shabi.  As can be seen with the test results, the bots are catching VVL errors again.\n\nHow to land this?  Do I suppress the new test failures and try to find which CLs introduced the errors?",
      "revId": "b168f48f369625cd176836ad0d5d2fdeca187b9b",
      "serverId": "3ce6091f-6c88-37e8-8c75-72f92ae8dfba"
    }
  ]
}